cli/internal/run/run.go

```diff
@@ -824,7 +824,7 @@ func (r *run) executeDryRun(ctx gocontext.Context, engine *core.Scheduler, g *co
 			Package:      pt.PackageName,
 			Hash:         hash,
 			Command:      command,
-			Dir:          pt.Pkg.Dir,
+			Dir:          pt.Pkg.Dir.ToString(),
```

Comment:
(request for change)
```suggestion
			Dir:          pt.Pkg.Dir.ToStringDuringMigration(),
```

Reply:
This is a JSON representation, not something we use for further computation. I think `.ToString()` is correct.